src/avio.jl

```diff
@@ -1017,7 +1017,7 @@ const CAMERA_DEVICES = String[]
 const DEFAULT_CAMERA_DEVICE = Ref{String}()
 const DEFAULT_CAMERA_OPTIONS = AVDict()
 
-function get_camera_devices(ffmpeg, idev, idev_name)
+function get_camera_devices(idev, idev_name)
```

Comment:
```suggestion
get_camera_devices(ffmpeg, idev, idev_name) = get_camera_devices(idev, idev_name)

function get_camera_devices(idev, idev_name)
```

I am not sure anyone uses this, but this will avoid breakage if so.

Reply:
I agree we should add this but in a place where have deprecated apis that we eventually intend to remove.

Reply:
```suggestion
get_camera_devices(ffmpeg, idev, idev_name) = get_camera_devices(idev, idev_name)
@deprecate get_camera_devices(ffmpeg, idev, idev_name) get_camera_devices(idev, idev_name)

function get_camera_devices(idev, idev_name)
```

Should work.